app/models/response/emcsTfe/draftMovement/DraftId.scala

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.response.emcsTfe.draftMovement
+
+import play.api.libs.json._
+
+case class DraftId(value: String)
+
+object DraftId {
+
+  implicit val reads: Reads[DraftId] = for {
+    draftId <- (__ \ "draftId").read[String]
+  } yield {
+    DraftId(draftId)
+  }
```

Comment:
I think this could be written as this if you wanted it as one line - but the for comprehension might be more human readable anyway
```suggestion
  implicit val reads: Reads[DraftId] = 
    (__ \ "draftId").read[String].map(DraftId(_))
```